### PR TITLE
Added tests for plasma ion population properties related to LTE mode

### PR DESCRIPTION
**Pull request by aoifeboyle**
This PR includes two tests. The first tests the calculation of 'phi' for an LTE plasma. The second checks that the ion populations (number densities) have been converged correctly, by checking that the electron density calculated from the ion populations is close to that obtained from the Saha equation. I'm focusing on testing the properties relevant to a simple LTE plasma for now because I'd like to be able to start working on that soon.

### CODE REVIEW

File: tardis/plasma/tests/test_property_ion_population.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+from tardis.plasma.properties.ion_population import (PhiSahaLTE,
+IonNumberDensity)
+from tardis.plasma.properties.general import (BetaRadiation, GElectron,
+NumberDensity)
+from tardis.plasma.properties.partition_function import (LevelBoltzmannFactor,
+LTEPartitionFunction)
+from tardis.plasma.properties.atomic import Levels, AtomicMass, IonizationData
+
+def test_phi_saha_lte(t_rad, included_he_atomic_data):
+    beta_rad_module = BetaRadiation(None)
```
Comment (wkerzendorf) [inline tardis/plasma/tests/test_property_ion_population.py]: you could make these things fixtures - if you need them more often.

Comment (aoifeboyle) [inline tardis/plasma/tests/test_property_ion_population.py]: You mean beta_rad, g_electron etc.?

Comment (wkerzendorf) [inline tardis/plasma/tests/test_property_ion_population.py]: yep. you also define fixtures in the `conftest.py` in the directory above. then they will be available for all functions.

Comment (aoifeboyle) [inline tardis/plasma/tests/test_property_ion_population.py]: Ok great - will do!

**Comment by wkerzendorf** [comment]
that looks fairly complete - do we merge this?

**Comment by aoifeboyle** [comment]
I think so! But if you want to wait and get someone else to check it that's fine.

**Comment by aoifeboyle** [comment]
I just closed the PR by accident and re-opened it again - sorry :P

**Pull request merged**